ArcGISRuntimeSDKQt_CppSamples/Routing/FindRoute/FindRoute.cpp

```diff
@@ -143,35 +143,13 @@ void FindRoute::setupRouteTask()
     if (loadStatus == LoadStatus::Loaded)
     {
       // Request default parameters once the task is loaded
-      m_routeTask->createDefaultParameters();
+      m_routeTask->createDefaultParametersAsync().then(this, [this](const RouteParameters& routeParameters)
+      {
+        // Store the resulting route parameters
+        m_routeParameters = routeParameters;
+      });
     }
   });
-  //! [FindRoute new RouteTask]
```

Comment:
Comments like this are qdoc tags and are read by our doc build. If you delete them, our doc build will no longer be able to find this code and have problems. 

Any doc build tags like this should be repositioned around the new, updated code block that is relevant.

Reply:
Modified.